### ISSUE TITLE
xmrig does not dye if network is down

### ISSUE THREAD

**Issue by gaga9999** [xmrig/xmrig#2553]
is that intended or not, i dont know, it dies if one starts it without parameters, but it does not quit when network is down. an option would be fine for those who run it on watchdog and receive notfivications. exit would be a trigger for an alarm.

**Comment by Spudz76** [comment]
No, because then you lose your memory allocations, and it's better to leave it running until the network comes back, it doesn't "do" anything but try to reconnect during the no-network phase.

**Comment by gaga9999** [comment]
ok got it, recoverable error = keeps running forever.

**Comment by Spudz76** [comment]
That is a better way to describe it, yes.
